Documentation/infra-guide/arm64-infra.md

```diff
@@ -13,7 +13,7 @@ The infrastructure consists of two `c3.large.arm` bare metal servers kindly prov
 
 ## Infrastructure support
 
-The etcd project aims to self manage and resolve issues with project infrastructure internally where possible, however if situations emerge where we need to engage support from Equinix Metal we can open an issue under the [CNCF Community Infrastructure Lab](https://github.com/cncf/cluster/issues) project. If the situation is urgent contact @vielmetti directly who can provide further assistance or escalation points.
+The etcd project aims to self manage and resolve issues with project infrastructure internally where possible, however if situations emerge where we need to engage support from Equinix Metal we can open an issue under the [CNCF Community Infrastructure Lab] project. If the situation is urgent contact @vielmetti directly who can provide further assistance or escalation points.
```

Comment:
Could we get LGTM from @vielmetti ?

Reply:
Related comment for context https://github.com/etcd-io/etcd/issues/15952#issuecomment-1579486408

Reply:
If the situation warrants it I certainly want to be in the loop. For 24x7 support, though, you should reference https://deploy.equinix.com/support/ as there is a whole team available to address questions should they ever arise that need operational support of the system itself.

Reply:
Great suggestion @vielmetti thanks, added as an option 👍🏻